api/tests/Behat/features-v2/app-notification/super-admin-banner-visibility.feature

```diff
@@ -0,0 +1,21 @@
+@v2 @v2_admin @app-notification
+Feature: Restricting visibility of banner notification to super admin only
+    To ensure that I know what version of the app I am using
+    As a super admin user
+    I want to see a banner notification confirming the hosted environment I am logged in to
+
+    @super-admin
+    Scenario: A super admin has visibility of the banner notification in the admin app
+        Given a super admin user accesses the admin app
+        Then I should see a banner confirming the 'admin' version of the app I am using
```

Comment:
So this is going to be showing the name of the hosted environment (e.g. ddpb4333, integration, preproduction etc) rather than which view of the app you're looking at (e.g. frontend or admin). I think the best way to update this test will be to match on the text you expect but allow the name of the environment to be any string. Heres a good SO post on explaining the type of regex you could use for this https://stackoverflow.com/questions/12677178/regular-expression-with-wildcards-to-match-any-character